data/build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   compileSdkVersion 30
-  buildToolsVersion "29.0.2"
+  buildToolsVersion "29.0.3"
```

Comment:
Why is this change needed?

Reply:
ThebuildToolsVersion has been reverted back